lstchain/calib/camera/drs4.py

```diff
@@ -83,11 +84,15 @@ def _fill_pedestal_event_jit(waveform, expected_pixel_id, first_cap_array,
                             numped[gain, pixel, posads] += 1
 
     def finalize_pedestal(self):
-        if np.sum(self.numped==0) > 0:
-            raise RuntimeError("Not enough events to coverage all capacitor. "
-                               "Please use more events to create pedestal file.")
+        self.meanped = self.meanped / self.numped
+        pixels_with_nan_value = np.where(np.isnan(self.meanped).any(axis=0))
+        if len(pixels_with_nan_value[0]) > 0:
+            # Find failing pixels id
+            self.failing_pixels_list = np.unique(pixels_with_nan_value[0])
+            print("Failing pixels:")
+            print(self.failing_pixels_list)
         else:
-            self.meanped = self.meanped / self.numped
+            print("No pixels with nan values")
```

Comment:
maybe write  instead of "no pixels with nan values" something more descriptive: "no pixels with failing pedestal calculation", or even better just remove this print, it is enough that you show it when there is a problem

Reply:
Done